src/geometry/manifold/derivation_bundle.lean

```diff
@@ -0,0 +1,163 @@
+/-
+Copyright © 2020 Nicolò Cavalleri. All rights reserved.
```

Comment:
```suggestion
Copyright © 2021 Nicolò Cavalleri. All rights reserved.
```

Reply:
I actually already published this code last year in another repository! I don't know what that date should then be...

Reply:
Doesn't matter. 2020 is fine.